content/security/docs/iam.md

```diff
@@ -462,18 +464,121 @@ AWS_WEB_IDENTITY_TOKEN_FILE=/var/run/secrets/eks.amazonaws.com/serviceaccount/to
+
+This example S3 Bucket policy only grants access to objects in the S3 bucket it's attached to, only if `kubernetes-service-account`, `kubernetes-namespace`, `eks-cluster-arn` all meet their expected values, where the EKS cluster is hosted in the AWS account `111122223333`.
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::111122223333:root"
+            },
+            "Action": "s3:*",
+            "Resource":            [
+                "arn:aws:s3:::ExampleBucket/*"
+            ],
+            "Condition": {
+                "StringEquals": {
+                    "aws:PrincipalTag/kubernetes-service-account": "s3objectservice",
+                    "aws:PrincipalTag/eks-cluster-arn": "arn:aws:eks:us-west-2:111122223333:cluster/ProductionCluster",
+                    "aws:PrincipalTag/kubernetes-namespace": "s3datanamespace"
+                }
+            }
+        }
+    ]
+}
+```
+
+### EKS Pod Identities compared to IRSA
+
+Both EKS Pod Identities and IRSA are preferred ways to deliver temporary AWS credentials to your EKS pods. Unless you have specific usecases for IRSA, we recommend you use EKS Pod Identities when using EKS. This table helps compare the two features.
+
+|# |EKS Pod Identities | IRSA |
+|:--|:--|:--|
+|Requires permission to create an OIDC IDP in your AWS accounts?|No|Yes|
+|Requires unique IDP setup per cluster |No|Yes|
+|Sets relevant session tags for use with ABAC|Yes|No| 
+|Requires an iam:PassRole Check?|Yes| No |
+|Uses AWS STS Quota from your AWS account?|No|Yes|
+|Can access other AWS accounts | Indirectly with role chaining | Directly with sts:AssumeRoleWithWebIdentity|
+|Compatible with AWS SDKs |Yes|Yes|
+|Requires Pod Identity Agent Daemonset on nodes? |Yes|No|
+
+## Identities and Credentials for EKS pods Recommendations
 
 ### Update the aws-node daemonset to use IRSA
 
 At present, the aws-node daemonset is configured to use a role assigned to the EC2 instances to assign IPs to pods.  This role includes several AWS managed policies, e.g. AmazonEKS_CNI_Policy and EC2ContainerRegistryReadOnly that effectively allow **all** pods running on a node to attach/detach ENIs, assign/unassign IP addresses, or pull images from ECR. Since this presents a risk to your cluster, it is recommended that you update the aws-node daemonset to use IRSA. A script for doing this can be found in the [repository](https://github.com/aws/aws-eks-best-practices/tree/master/projects/enable-irsa/src) for this guide.
 
+The aws-node daemonset does not support EKS Pod Identities at this time.
```

Comment:
Can we put a date down? I'm pretty sure this work is in-flight, so I'd rather timestamp this since it won't be true very soon